docs/examples/lipstick-pig.md

```diff
@@ -0,0 +1,315 @@
+In this example we'd like to discuss the effectiveness of debiasing techniques in word embeddings.
+This example is heavily inspired by the [lipstick on a pig paper](https://arxiv.org/pdf/1903.03862.pdf)
+by Hila Gonen and Yoav Goldberg.
+
+## Meaning
+
+In word embedding space you might wonder if a direction in embedding space might represents meaning.
+
+- $v_{man}$ represents the word embedding for "man" .
+- $v_{woman}$ represents the word embedding for "woman" .
+- You could argue that the axis $v_{man} - v_{woman}$ might represent "gender".
+
+One side of the "gender" axis would represent the male gender while the other end represents the female gender. This spectrum might allow you to guess if words are more "male" or more "female". It can also be used as a measurement for bias in language.
+
+## Similarity
+
+There are some axes that we might come up with that should be similar to this gender axis.
+
+- $v_{he} - v_{she}$
+- $v_{king} - v_{queen}$
+- $v_{brother} - v_{sister}$
+
+It would be highly unfortunate though if the following pairs of words would display similarity.
+
+- $v_{nurse} - v_{physician}$
+- $v_{nurse} - v_{surgeon}$
+- $v_{nurse} - v_{doctor}$
+
+Being a woman should not imply that you are a nurse just like that being a man should not imply that you are a surgeon. It'd be a shame if we were using embeddings where such stereotypical bias present. Unfortunately, it's likely in the embeddings. Historically woman have not gotten the same opportunities as men. This is bound to be reflected on websites like Wikipedia which are a common source of data to train word embeddings. So let's make a similarity chart to confirm if this is the case.
```

Comment:
1. "Being a woman should not imply that you are a nurse just like that being a man should not imply that you are a surgeon." --> isn't the opposite way of saying this the better: "Being a nurse should not imply that you are a woman..." because that's what we call the bias in our thinking, not the reverse?!

2. ...such stereotypical bias **is** present.
3. Historically**,**
4. woman --> women (after historically)